src/Pages/login.tsx

```diff
@@ -1,23 +1,44 @@
-import { ChangeEvent, useState } from "react";
-import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import { ChangeEvent, FormEvent, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
 
 const Login = () => {
-    const [username, setUserName] = useState<string>('');
+    const [email, setEmail] = useState<string>('');
     const [password, setPassword] = useState<string>('');
+    const [incorrectRes, setIncorrectRes] = useState<boolean>(false);
```

Comment:
@ShiranthaKellum, the variable name is confusing, try to use meaningful variable names,

Reply:
Done.